Description: Upstream changes introduced in version 0.120-r1-4
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 q4wine (0.120-r1-4) unstable; urgency=low
 .
   * Updated debian/patches/
     - additional fix for Debian GNU/Hurd
   * Edited debian/control:
     - section with dependencies of binary package was updated:
       + Ubuntu users now need to have wine, wine1.0, wine1.2 or wine1.3
         (Closes: #614357)
 .
 The person named in the Author field signed this changelog entry.
Author: Boris Pek <tehnick-8@mail.ru>
Bug-Debian: http://bugs.debian.org/614357

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- q4wine-0.120-r1.orig/src/q4wine-lib/q4wine-lib.cpp
+++ q4wine-0.120-r1/src/q4wine-lib/q4wine-lib.cpp
@@ -55,11 +55,8 @@ QList<QStringList> corelib::getWineProce
     // Check for /proc directory exists
     QDir dir("/proc");
     if (!dir.exists()){
-        if (this->showError(message, false) == QMessageBox::Ignore){
-            procline << "-1";
-            proclist << procline;
-            return proclist;
-        }
+        proclist.clear();
+        return proclist;
     }
 
     /* On Linux:
